### PR TITLE
Workflow Catalog synchronisation

### PR DESCRIPTION
**Pull request by laurianed**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
```
Comment (fviale) [inline src/docs/user/ProActiveUserGuide.adoc]: This title is a bit long, why not "Importing/Exporting Workflows in the Catalog" ?

Then you can detail in the introduction the multiple options you have available for importing/exporting workflows

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
```
Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: from the selected bucket --> from the current bucket

Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: you've mentioned two times bucket in your previous sentence, so "This bucket" is which bucket? better to use "which is ..."

File: src/docs/user/ProActiveUserGuide.adoc [hunk 3]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+
+==== Send Workflows directly
+
+If the Workflows to share are sent to a Scheduler having a public API, or to another bucket of the local Scheduler, it is convenient to send them directly.
+
```
Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: It is convenient to send the Workflows to be shared directly, if they are sent to the other buckets on the same Scheduler or to a Scheduler with a public API:

File: src/docs/user/ProActiveUserGuide.adoc [hunk 4]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
+
+==== Export Workflows into an archive, then Import them
+
+If the remote Scheduler is running in another network, of if you cannot use direct sending for some reason, you can export Workflows into an archive, that you can transfer by any file transfer method (email, usb key, etc.) and then import this archive into the other Scheduler.
```
Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: "in another network" is strange to me, but I don't know how to change it. 

of if you cannot ... --> or the direct sending can not be used for some reason, 

that you can transfer --> which can be transferred ... (), and then import the archive to the Scheduler you want to use.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 5]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+
+To export the selected Workflows as a ZIP archive, select a bucket, then the targeted Workflows. Choose "Export selected Workflows" into the list and click on `OK`. You'll get a ZIP file that you'll be able to tranfer to the machine on which the other Scheduler runs.
```
Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: that you'll be able .... --> which can be transferred to ....

File: src/docs/user/ProActiveUserGuide.adoc [hunk 6]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+
+To import the archive on this other Scheduler, select the bucket to which the Workflows will be added. Then choose `Import archive of Workflows` and click on `OK`.
+
```
Comment (sophiesongge) [inline src/docs/user/ProActiveUserGuide.adoc]: "this other Scheduler" sounds strange to me

File: src/docs/user/ProActiveUserGuide.adoc [hunk 7]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
```
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: Globally, I would rather separate the description from the instructions. For instance, export/import section may have a global description and then separate subsections.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 8]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
+
```
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: [...] it is possible to send Workflows from a bucket to another.
The destination bucket can belong to the [...]

File: src/docs/user/ProActiveUserGuide.adoc [hunk 9]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+==== Send Workflows directly
+
+If the Workflows to share are sent to a Scheduler having a public API, or to another bucket of the local Scheduler, it is convenient to send them directly.
```
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: [...] are sent to a bucket within the same local Scheduler or to any Scheduler having a public API, it is [...]

File: src/docs/user/ProActiveUserGuide.adoc [hunk 10]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
+
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
```
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: First, select the bucket and the Workflows to share, then choose [...]

File: src/docs/user/ProActiveUserGuide.adoc (hunk 4 above)
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: [...] _or_ if you cannot [...]
[...] into the desired Scheduler.

File: src/docs/user/ProActiveUserGuide.adoc (hunk 5 above)
Comment (vinseon) [inline src/docs/user/ProActiveUserGuide.adoc]: To be consistent with the previous instruction, I would start by:

First, select the bucket and the Workflows to export, then choose [...]

IMHO `that you'll be able to tranfer [...]` is confusing, I would simply put `that can be then imported on another Scheduler.` for example.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 11]
```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
...
 
+=== Importing/Exporting Workflows into the Catalog
+
+In the Workflow Catalog, it is possible to select Workflows from the current bucket and send them to another bucket, which can belong to the same Catalog or to another Scheduler's Catalog.
+
+==== Send Workflows directly
+
+It is convenient to send the Workflows to share directly, if they are sent to a Scheduler with a public API, or to another bucket of the local Scheduler.
+
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
```
Comment (fviale) [inline src/docs/user/ProActiveUserGuide.adoc]: You don't describe the difference in the procedure between sending to another scheduler or to the same scheduler on another bucket. Is there not an url you must enter ? In that case, you could maybe provide an example of the two use cases.

**Pull request merged**